test/CDFDocumentTest.php

```diff
@@ -65,7 +65,7 @@ public function testGetEntity($objectToAdd, $notAddedObject)
   public function testGetEntities($objectOne, $objectTwo)
   {
     //Check getting added and not added Object
-    $this->cdfDocument->setCDFEntities($objectOne, $objectTwo);
+    $this->cdfDocument->setCdfEntities($objectOne, $objectTwo);
 
     foreach ($this->cdfDocument->getEntities() as $entity) {
       $this->assertInstanceOf(CDFObject::class, $entity);
```

Comment:
Should CDFObject also be changed?

Reply:
Out of scope. Please see my overall comment on this PR.